safe/gui/tools/help/osm_downloader_help.py

```diff
@@ -98,17 +98,19 @@ def content():
     message.add(tips)
 
     message.add(m.Paragraph(
-        tr('When the'),
-        m.ImportantText(tr(' Political boundaries ')),
-        tr(
+        # format 'When the __Political boundaries__' for proper i18n
+        tr('When the %s '
             'box in the Feature types menu is ticked, the Political boundary '
             'options panel will be enabled. The panel lets you select which '
             'admin level you wish to download. The admin levels are country '
             'specific. When you select an admin level, the local name for '
             'that admin level will be shown. You can change which country '
             'is used for the admin level description using the country drop '
             'down menu. The country will be automatically set to coincide '
-            'with the view extent if a matching country can be found.')))
+            'with the view extent if a matching country can be found.') %
+        (
+            m.ImportantText(tr('Political boundaries')).to_html(),
```

Comment:
But if we do this we are breaking separation between content and markup - now we will html embedded in the string....I think that is bad...